chemprop/cli/train.py

```diff
@@ -535,7 +535,9 @@ def validate_train_args(args):
     if args.tracking_metric not in valid_tracking_metrics:
         raise ArgumentError(
             argument=None,
-            message=f"Tracking metric must be either 'val_loss' or one of the metrics specified via `--metrics`. Got {args.tracking_metric}",
+            message=f"Tracking metric must be one of {','.join(valid_tracking_metrics)}. "
+            f"Got {args.tracking_metric}. Additional tracking metric options can be specified with "
+            "the --metrics flag.",
```

Comment:
This suggestion is to make the message all on one line, is that right? I prefer to split the string over multiple lines so that each line isn't too long. But happy to change that to one line if you'd like.

Reply:
I am find to keep multiple lines here. The reason I made this suggestion is because the rest of the help messages in this module are written in one line.